client/js/analysisui/optionsview.js

```diff
@@ -69,14 +72,14 @@ var OptionsView = Backbone.View.extend({
     createLayoutView_group: function(level, item) {
 
         var name = item.name;
-        var format = item.format;
-        if (!format)
-            format = "list";
+        var style = item.style;
+        if (!style)
```

Comment:
good practice is to put spaces either side of an exclamation:

    if ( ! style) {

it can be easy to miss a !